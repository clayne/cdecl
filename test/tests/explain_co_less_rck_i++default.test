cdecl @ @ -xc++ @ struct S; explain int operator<(S const&) const = default @ 65
