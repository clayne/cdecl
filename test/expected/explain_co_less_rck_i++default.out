explain int operator<(S const&) const = default
        ^
9: error: operator < must return bool
